### PR TITLE
Update the settings page.

### PR DESCRIPTION
**Pull request by donnchawp**
Adds cards and teaser for the Crowdsignal Forms plugin.

Apply the patch and go to Settings->Crowdsignal where you will see the new settings page.

### CODE REVIEW

**Comment by digitalwaveride** [comment]
Thanks so much Donncha! We are close ;) 


I started with a setup that had both plugins installed.

When I got to Settings > Crowdsignal I am seeing this:

image

Unfortunately though clicking on "Advanced Settings" does nothing for me at that point.
This link is attached:
http://localhost:8000/wp-admin/options-general.php?page=crowdsignal-settings#


The setup with just the Classic Plugin installed shows this as expected 👍 :

image

Teaser etc. all good.

Unfortunately the "Advanced Settings" link did not work as well.

File: partials/html-admin-setup-footer.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ *
+ * @package Crowdsignal_Forms\Admin
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+?>
+
+</div>
```
Comment (CGastrell) [inline partials/html-admin-setup-footer.php:13]: ooof, closing tags from all over the world, harsh one

**Review by CGastrell — APPROVED** [review]
Don't know about the little details, but this is looking good for me :)

File: partials/html-admin-setup-footer.php (hunk 1 above)
Comment (donnchawp) [reply partials/html-admin-setup-footer.php]: I just indented everything and I think fixed the div tags.

**Review by digitalwaveride — APPROVED** [review]
Thanks so much Donncha !!! 🚀 🚀 🚀

**Pull request merged**